docs/resources/Guild.md

```diff
@@ -252,15 +252,19 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
-| roles          | array of snowflakes                             | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids                                                           |
-| joined_at      | ISO8601 timestamp                               | when the user joined the guild                                                                                             |
-| premium_since? | ?ISO8601 timestamp                              | when the user started [boosting](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-) the guild |
-| deaf           | boolean                                         | whether the user is deafened in voice channels                                                                             |
-| mute           | boolean                                         | whether the user is muted in voice channels                                                                                |
+| Field           | Type                                            | Description                                                                                                                |
+| --------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| user?           | [user](#DOCS_RESOURCES_USER/user-object) object | the user this guild member represents                                                                                      |
+| nick            | ?string                                         | this users guild nickname                                                                                                  |
+| roles           | array of snowflakes                             | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids                                                           |
+| joined_at       | ISO8601 timestamp                               | when the user joined the guild                                                                                             |
+| premium_since?  | ?ISO8601 timestamp                              | when the user started [boosting](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-) the guild    |
+| deaf            | boolean                                         | whether the user is deafened in voice channels                                                                             |
+| mute            | boolean                                         | whether the user is muted in voice channels                                                                                |
+| hoisted_role?\* | ?string                                         | role id that represents the member's hoisted role in the guild                                                             |
```

Comment:
The proper type for ID fields is `snowflake`.

Reply:
Solved in 165eb4c598fa9f6a8fcad0435f584625502e1c54